src/components/Calculator.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import Button from './Button';
+import '../stylesheets/calculator.scss';
+
+const Calculator = (props) => {
+  const { screen } = props;
+  return (
+    <div id="calculator">
+      <div id="screen">{screen}</div>
+      <div className="row">
+        <Button content="AC" />
+        <Button content="+/-" />
+        <Button content="%" />
+        <Button content="/" operator="true" />
+      </div>
+      <div className="row">
+        <Button content="7" />
+        <Button content="8" />
+        <Button content="9" />
+        <Button content="X" operator="true" />
+      </div>
+      <div className="row">
+        <Button content="4" />
+        <Button content="5" />
+        <Button content="6" />
+        <Button content="+" operator="true" />
+      </div>
+      <div className="row">
+        <Button content="1" />
+        <Button content="2" />
+        <Button content="3" />
+        <Button content="-" operator="true" />
+      </div>
+      <div className="row">
+        <Button content="0" operator="true" double="true" />
+        <Button content="." />
+        <Button content="=" operator="true" />
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ ] This component should be `functional` for a moment

Reply:
- [ ] Make sure all the boilerplate that got created by the CRA (texts, images, and styles) are deleted.

Reply:
@DeliceLydia Hi, I now deleted the remaining boilarplate but the components are already of the functional type, I don't understand 😕

Reply:
@Movgto Currently you used arrow functions but requirements say the component should be functional
like
```
function Calculator() {
}
```

Reply:
Thank you, I now just changed it to a functional component, do I need to request another review? @DeliceLydia